base_layer/wallet/src/output_manager_service/handle.rs

```diff
@@ -335,6 +393,26 @@ impl OutputManagerHandle {
         }
     }
 
+    pub async fn add_rewindable_unvalidated_output(
```

Comment:
I see this method is not even used in tests, I think it should come out

Reply:
Yeah, still busy with that one. That functionality needs to be tested though.